pages/nfttest/mint/_iscnId.vue

```diff
@@ -118,6 +118,7 @@ export default class NFTTestMintPage extends Vue {
   iscnOwner: string = ''
   iscnData: any = null
   apiData: any = null
+  ogImageUrl = decodeURIComponent(this.$route.query.ogImageUrl as string) || ''
```

Comment:
You don't need to call `decodeURIComponent ` for query if you just passed it plainly, vue-router would handle that